view/popup.js

```diff
@@ -144,27 +121,37 @@ startInspect.addEventListener("click", function () {
     return;
   }
   start();
-  chrome.runtime.getBackgroundPage((background) => {
-    background.start(url);
-  });
+  chrome.runtime.sendMessage({callMethod: 'start'}, (response)=> {});
 });
 startInspect.addEventListener("mouseover", () => setClasses("startInspectInfo", "badge badge-focus"));
 startInspect.addEventListener("mouseout", () => setClasses("startInspectInfo", "badge"));
 
 nowInspect.addEventListener("click", function () {
-  start();
-  chrome.runtime.getBackgroundPage((background) => {
-    background.start('');
+  chrome?.storage?.local.get(['inspectList'],(result)=>{
+
+    if(result?.inspectList?.length > 0)
+    {
+      const response = confirm("There are some steps already captured once you start inspecting it will clear eariler steps.?");
+      if(response)
+      {
+        start();
+        $('#inspect_table').remove();
+        chrome.runtime.sendMessage({callMethod: 'start'}, (response)=> {});
```

Comment:
please create constants for common literals, like `'start'`, `'stop'`, `'click'`, etc.

Reply:
Done with this change and pushed